CONTRIBUTING.rst

```diff
@@ -87,21 +87,38 @@ Ready to contribute? Here's how to set up `dj-stripe` for local development.
    those tests, the --skip-utc command line option is available on runtests.py.::
 
     $ pip install tox
+      then either
     $ tox
+      to test all environments defined in the tox.ini file, or 
+    $ tox -e py35
+      to test using only python3.5 version and environment 
 
 7. If your changes altered the models you may need to generate Django migrations::
 
     $ python makemigrations.py
 
-8. Commit your changes and push your branch to GitHub::
+8. If you'd like to package the module for use in other projects, or for integration
+  testing purposes:
+
+    $ python setup.py build;
+    $ python -m pip install --upgrade setuptools wheel; 
+    $ python setup sdist bdist_wheel;
+    $ ls dist/dj-stripe*;
+
+9. If you simply need to conduct integration testing while also developing, modifying, or otherwise extending the dj-stripe module in parallel, pip install the absolute path to the root directory from within your other project:
+   
+    $ cd ~/git/myproject/
+    $ pip install ../dj-stripe 
```

Comment:
I'm not sure this needs to be in the project readme.  Also note that `pip install -e` will handle both of these situations.

eg.

in a requirements.txt:

```
-e git+git@github.com:prismaticd/dj-stripe.git@cc92fddf63a5f5028476e8a06084831803dc0855#egg=dj_stripe
```

or on the command line (inside your project's virtualenv).

```
pip install -e ../dj-stripe
```